Push request for the Negotiation Mastery Program conversion to YAML file

Computer8543 opened the pull request.

jimmanzzullo reviewed (commented):
@michael-basil   I find the "Spiral Dynamics Integral..."article complementary to the "Spiral Dynamic Explained" video that is present in the green level.  I did notice there was redundancy is the material and more or less felt I was skimming that material rather than absorbing new insights into Spiral Dynamics itself.  That being said, I did appreciate the similarity and contrast between Maslow's model and Graves Model.  If the Spiral Dynamics Model is going to be present through all belt levels, I would consider if this addition or red will complement or flow into what may be eventually present in black.

This PR is for two articles, so I am reviewing it for the sake of Spiral Dynamics and not for the article "A Definitive Guide to Understanding Intrinsic Motivation"  I will read it, but wanted to put my thoughts down Spiral Dynamics (per our convo).  

P.S.  The label type for "A Definitive Guide to Understanding Intrinsic Motivation" is video-camera.  That may be incorrect as its an article?

michael-basil commented:
Hi @jimmanzzullo ,

Thanks for taking a look through this.

I believe the type designation is globe for both of the articles, but maybe I am looking at it incorrectly.

The idea is to revisit the model in red and have it handy in writing (versus video).

Additionally, since we spoke I added one more commit and link to the PR which is perhaps very impactful regarding motivation.

Have a great weekend!
🌿

billwestfall commented:
+1

jimmanzzullo reviewed (approved):
I have reviewed the articles and approve this pr.

Computer8543 committed:
Initial draft of Negotiation Mastery Program in a YAML file format.

Computer8543 committed:
Merge pull request #1 from Computer8543/kyle

Merge kyle branch into main branch

Computer8543 committed:
Merge Initial Draft code from main branch to begin negotiation. Also removed comments to make change so I can git commit directly to this issue.

Computer8543 committed:
Merge pull request #2 from Computer8543/main

Merge main branch changes into kyle branch

Computer8543 committed:
Merge branch 'mindset-dojo:main' into kyle

Computer8543 committed:
Create LICENSE.md

Add CC BY-SA 4.0 license to repo.

Computer8543 committed:
Create TODO.md

Create an initial draft of a minimal set of contribution guidelines for the Mindset Dojo repo.

Computer8543 committed:
Create README.md

Created README.md file to summarize the offerings of Mindset Dojo and it's community succinctly.

Computer8543 committed:
Merge pull request #3 from Computer8543/kyle

Merge kyle branch into main branch

Computer8543 committed:
Rename TODO.md to CONTRIBUTING.md

This is to follow open source naming conventions.

Computer8543 committed:
Merge pull request #4 from Computer8543/kyle

Rename TODO.md to CONTRIBUTING.md

Computer8543 committed:
Green Belt Push Request and Tag Change

Computer8543 committed:
Merge pull request #5 from Computer8543/kyle

Merge kyle branch into main

Computer8543 committed:
Delete README.md

Computer8543 committed:
Delete CONTRIBUTING.md

Computer8543 committed:
Delete LICENSE.md

Computer8543 committed:
Update kyle-ingersoll.yaml

Changed back to original yellow belt status and tag.

michael-basil commented on _data/negotiation-mastery-program.yaml:
```diff
@@ -0,0 +1,174 @@
+title: Negotiation Mastery Program
+mission: Awaken the negotiator within—calm, clear, and creative.
+principles:
```
Remove the "1st" "2nd" ... just use a list of strings with a label for each one called "label"

Like this ...

https://github.com/SAP-samples/dojo/blob/main/docs/_data/domains/Mindset.yaml

michael-basil commented on _data/negotiation-mastery-program.yaml:
```diff
@@ -0,0 +1,174 @@
+community:
+  - url: https://dojo.center/community
+practices:
+  - reflection:
```
reflection section is good ... but I want to talk through the distinction between the continuous reflection and belt reflection.

I sense there is some confusion here (not in this file), but when it comes up in conversation

michael-basil commented on _data/negotiation-mastery-program.yaml:
```diff
@@ -0,0 +1,174 @@
+      - In the future as a black belt, how will I show up and what sorts of things do I do?
+      - To this point, what have I learned so far overall and specifically to this belt level?
+      - How do I intend on applying what I have learned in the short and medium term?
+    - systems thinking:
```
Remove this for now

michael-basil commented on _data/negotiation-mastery-program.yaml:
```diff
@@ -0,0 +1,174 @@
+      - To this point, what have I learned so far overall and specifically to this belt level?
+      - How do I intend on applying what I have learned in the short and medium term?
+    - systems thinking:
+    - intentional conversation:
```
Remove this for now

michael-basil commented on _data/negotiation-mastery-program.yaml:
```diff
@@ -0,0 +1,174 @@
+    - intentional conversation:
+    - challenge:
+      - Every level has challenges to pass in order to prove sufficient learning and skill to proceed.
+    - run the gauntlet:
```
Remove this for now

michael-basil commented on _data/negotiation-mastery-program.yaml:
```diff
@@ -0,0 +1,174 @@
+  - Energetic
+  - Somatic
+  - Metaphorical
+belt level:
```
change this to "levels" ... also use underscores for any labels in YAML not spaces between

michael-basil commented on _data/negotiation-mastery-program.yaml:
```diff
@@ -0,0 +1,174 @@
+belt level:
+  - white:
+    - kyu: 7th
+    - principle: Orientation
```
this should be ... focus

michael-basil commented on _data/negotiation-mastery-program.yaml:
```diff
@@ -0,0 +1,174 @@
+  - Somatic
+  - Metaphorical
+belt level:
+  - white:
```
The levels are a bit wonky ... here is what white (7th kyu should be) ... note I am using negative for kyu and positive for dan ranks (so white is 7th kyu which is -7 and shodan - black is 1st dan which would be 1) ... but I only showed white here ...

`---
levels:
  - color: white
    level: -7
    principle: Orientation
    challenge: FEBI assessment
    exercises:
      - label: How to Start a Movement
        url: https://www.ted.com/talks/derek_sivers_how_to_start_a_movement
      - label: Focused Energy Balance Indicator (FEBI)
        url: https://zenleader.global/febi-offerings/
      - label: Focused Energy Balance (FEBI) Choice
        url: https://zenleader.global/s/FEBI-Choice-Article-Jan-2017-3p5x.pdf
      - label: Focused Energy Balance (FEBI) Technical Background
        url: https://zenleader.global/s/FEBI-Technical-Manual-2024.pdf
`

Computer8543 committed:
Update kyle-ingersoll.yaml

Changed Base 64 String to Central Repository String to resolve diff conflict in file.
